source/includes/administration/_saml_settings.md

```diff
@@ -6,13 +6,18 @@ Allows to change the configuration for the service provider integrated with SAML
 
 #### List SAML settings
 
-`GET /saml_settings`
+`GET /saml_settings?organization_id=:organization_id`
 
 ```shell
 # Retrieve SAML settings
-curl "https://cloudmc_endpoint/api/v1/saml_settings" \
+curl "https://cloudmc_endpoint/api/v2/saml_settings?organization_id=:organization_id" \
```

Comment:
Do we keep both endpoints or only v2?

Reply:
TBH I'm not sure. @jasondias9 would you happen to know?

Reply:
I tested on v1, all looks good 👍

Reply:
We should keep it in v1 imo

Reply:
Yeah looks like thats the move. Sorry for the confusion.